Update the game over message

james-campanella opened the pull request:
Update the game over message so people know how to play again!

Commit:
Update game over message

Commit:
Update to 1 in STEP and README.md

Commit:
Update to 2 in STEP and README.md

Commit:
Update to 3 in STEP and README.md

james-campanella reviewed (commented):
This change should add clarification to how to play again and is safe for merge.

Commit:
Update to 4 in STEP and README.md

steaminghotjava commented on index.html:
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

steaminghotjava replied:
Are the emoji's really needed?  Suggested change to remove them.

Commit:
Update to 5 in STEP and README.md

steaminghotjava committed:
Update index.html

Removed the emojis from the refresh to play again portion of the game over message.  They did not add much and might be confusing to screen readers.

Commit:
Update to 6 in STEP and README.md
